src/components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import Radio from './index'
+
+const stories = {
+  component: 'Radio',
+  props: [
+    {
+      name: 'checked',
+      type: 'boolean',
+      default: 'false',
+      description: 'Enables the checked option of the radio'
+    },
+    {
+      name: 'color',
+      type: 'StatusColor',
+      default: 'primary',
+      description: 'The color of the radio'
+    }
+  ],
+  stories: [
+    {
+      name: 'Radio',
+      description: 'Simple Radio',
+      render: <Radio> </Radio>,
+      prop: '',
+      code: `
+        <Radio>
+        </Radio>
+      `
+    },
+    {
+      name: 'Radio Checked',
+      description: 'Radio with checked prop',
+      render: <Radio checked> </Radio>,
+      prop: 'checked',
+      code: `
+        <Radio checked>
+        </Radio>
```

Comment:
The Radio component does not have children prop, so can be closed with only 1 tag: `<Radio checked />`